src/framework/tooltip/Tooltip.jsx

```diff
@@ -0,0 +1,134 @@
+import React, {
+  Component,
+  PropTypes,
+} from 'react';
+
+import ReactDOM from 'react-dom';
+
+import {
+  Box,
+  Text,
+} from '../../framework/framework';
+
+export default class Tooltip extends Component {
+  constructor(props) {
+    super(props);
+
+    this.content = this.props.message;
+    this.children = this.props.children;
+    this.onMouseEnter = this.onMouseEnter.bind(this);
+    this.onMouseLeave = this.onMouseLeave.bind(this);
+  }
+
+  componentDidMount() {
+    this.tooltip = this.refs.tooltipPosition;
```

Comment:
Little confusing that `this.tooltip` is actually not the element with ref `tooltip` but `tooltipPosition`.